app/controllers/pageflow/editor/files_controller.rb

```diff
@@ -81,7 +90,7 @@ def destroy
       private
 
       def create_params
-        file_attachment_params
+        file_params.permit(:file_name)
```

Comment:
Paperclip actually doesn't sanitize the file name. I added a validation on `HostedFile` and a spec for the `FilesController` to prevent file_name-spoofing